examples/subprocess_shell.py

```diff
@@ -31,3 +34,21 @@ def Popen(*args, **kwargs):
 subprocess.Popen(command, shell=True)
 
 subprocess.Popen('/bin/ls && cat /etc/passwd', shell=True)
+
+# Issue #157
```

Comment:
You can remove references to bugs/issues.  We can always use git or git blame to look up changes based on what issue.

Reply:
Done